pyfar/classes/audio.py

```diff
@@ -357,6 +359,14 @@ def __pow__(self, data):
     def __rpow__(self, data):
         return power((data, self), 'time')
 
+    def __matmul__(self, data):
+        return matrix_multiplication(
+            (self, data), 'time', axes=[(-2, -1), (-2, -1), (-2, -1)])
```

Comment:
We would not have to specify `axes` here (and below), since we are using the default. We could of course if we want to be explicit.